### DISCUSSION TITLE
Cbc can be installed with conda

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/pulp#431] [Show and tell]
I just wanted to alert folks to the fact that there is an officially supported conda package for Cbc that can be installed on Linux and OSX with `conda create -n cbc coin-or-cbc -c conda-forge`. This will be kept up-to-date with the latest releases of Cbc. It could be useful for PuLP to check whether Cbc has been conda installed and to use that version automatically if it is present.

**Comment by stumitchell** [comment]
Well the CBC_CMD solver exists for this purpose it will just use any CBC in
the path. The default solver doesn't have the smarts to work out if there
is cbc already installed (and a higher version number) but that is a
feature that can be added I guess.

Will probably make the jupyter notebook issues a bit easier to handle.

Stu

Stuart Mitchell
PhD Engineering Science
Extraordinary Freelance Programmer and Optimisation Guru
www.stuartmitchell.com

**Reply by tkralphs** [reply]
To follow up on this, once you activate the conda environment, the Cbc executable is automatically in the `PATH`, so this should work fine. But yes, there might have to be some version checking. Isn't version checking already there, though? This would be necessary for `CBC_CMD` in any case, right?
